### PR TITLE
V1.10.0 small items

### PR DESCRIPTION
**Pull request by index-git**
Per commit.

### CODE REVIEW

File: src/layman/common/util.py [hunk 1]
```diff
@@ -12,3 +12,12 @@ def merge_infos(infos):
         for (name, info) in source.items():
             result_infos[name].update(info)
     return result_infos
+
+
+def clear_publication_info(info):
```
Comment (jirik) [inline src/layman/common/util.py:17]: Test that GET Layer|GET Map responses do not contain id and type keys. E.g. in src/layman/layer/rest_test.py, line 312
```
layer_info = client.get(url_for('rest_layer.get', username=username, layername=layername)).get_json()
```

File: src/layman/common/prime_db_schema/migrate_test.py [hunk 2]
```diff
@@ -50,7 +50,9 @@ def test_schema():
                       settings.RIGHTS_EVERYONE_ROLE)
 
         workspaces = run_query(f'select count(*) from {DB_SCHEMA}.workspaces;')
-        assert workspaces[0][0] == len(util.get_usernames())
+        assert workspaces[0][0] == len(util.get_workspaces())
+        workspaces = run_query(f'select count(*) from {DB_SCHEMA}.users;')
```
Comment (jirik) [inline src/layman/common/prime_db_schema/migrate_test.py:54]: rename workspaces to users

File: src/geoserver/epsg_properties.py [hunk 3]
```diff
@@ -24,13 +24,14 @@
...
           '"6326"]],PRIMEM["Greenwich",0,AUTHORITY["EPSG","8901"]],UNIT["degree",0.0174532925199433,AUTHORITY["EPSG","9122"]],'
           'AUTHORITY["EPSG","4326"]]',
-    5514: '5514=PROJCS["S-JTSK / Krovak East North",GEOGCS["S-JTSK",DATUM["System_Jednotne_Trigonometricke_Site_Katastralni",'
-          'SPHEROID["Bessel 1841",6377397.155,299.1528128,AUTHORITY["EPSG","7004"]],TOWGS84[589,76,480,0,0,0,0],AUTHORITY["EPSG",'
-          '"6156"]],PRIMEM["Greenwich",0,AUTHORITY["EPSG","8901"]],UNIT["degree",0.0174532925199433,AUTHORITY["EPSG","9122"]],'
-          'AUTHORITY["EPSG","4156"]],PROJECTION["Krovak"],PARAMETER["latitude_of_center",49.5],PARAMETER["longitude_of_center",'
-          '24.83333333333333],PARAMETER["azimuth",30.28813972222222],PARAMETER["pseudo_standard_parallel_1",78.5],'
-          'PARAMETER["scale_factor",0.9999],PARAMETER["false_easting",0],PARAMETER["false_northing",0],UNIT["metre",1,'
-          'AUTHORITY["EPSG","9001"]],AXIS["X",EAST],AXIS["Y",NORTH],AUTHORITY["EPSG","5514"]]',
+    5514: '5514=PROJCS["S-JTSK / Krovak East North",GEOGCS["S-JTSK",DATUM["System Jednotne Trigonometricke Site Katastralni",'
```
Comment (jirik) [inline src/geoserver/epsg_properties.py:27]: @jirik Add 5514-related WFS test
Consider also tests for other EPSG codes

**Review by jirik — CHANGES_REQUESTED** [review]

File: deps/geoserver/sample/geoserver_data/user_projections/epsg.properties [hunk 4]
```diff
@@ -0,0 +1 @@
+5514=PROJCS["S-JTSK / Krovak East North",GEOGCS["S-JTSK",DATUM["System Jednotne Trigonometricke Site Katastralni",SPHEROID["Bessel 1841",6377397.155,299.1528128,AUTHORITY["EPSG","7004"]],TOWGS84[572.213,85.334,461.94,4.9732,-1.529,-5.2484,3.5378],AUTHORITY["EPSG","6156"]],PRIMEM["Greenwich",0.0,AUTHORITY["EPSG","8901"]],UNIT["degree",0.017453292519943295],AXIS["Geodetic longitude",EAST],AXIS["Geodetic latitude", NORTH],AUTHORITY["EPSG","4156"]],PROJECTION["Krovak",AUTHORITY["EPSG","9819"]],PARAMETER["latitude_of_center",49.5],PARAMETER["longitude_of_center",24.833333333333332],PARAMETER["azimuth", 30.288139722222223],PARAMETER["pseudo_standard_parallel_1",78.5],PARAMETER["scale_factor",0.9999],PARAMETER["false_easting",0.0],PARAMETER["false_northing",0.0],UNIT["m", 1.0],AXIS["X",EAST],AXIS["Y",NORTH],AUTHORITY["EPSG","5514"]]
```
Comment (index-git) [inline deps/geoserver/sample/geoserver_data/user_projections/epsg.properties:1]: I think, it is better to have this definition on only one place and this is the second. I would love to hear the reasons.

**Review by jirik — APPROVED** [review]

**Pull request merged**